solutions/validation/validate-pre-install.py

```diff
@@ -39,27 +56,37 @@
 ose_package_not_installed_dict = {}
 ssh_connection = HandleSSHConnections()
 selinux_dict = {}
-ose_repos = ["rhel-7-server-rpms", "rhel-7-server-extras-rpms", "rhel-7-server-ose-3.1-rpms"]
+if options.openshift_version:
+    if "3.2" in options.openshift_version:
+        openshift_server_repo = "rhel-7-server-ose-3.2-rpms"
+    elif "3.1" in options.openshift_version:
+        openshift_server_repo = "rhel-7-server-ose-3.1-rpms"
+    elif "3.3" in options.openshift_version:
```

Comment:
@etsauer do we want to just remove the version check and just have it apply what the user provides? Makes maintainability easier instead of having to update scripts. It will fail down the line anyways if it is incorrect